### PR TITLE
shed: Util for creating ID CIDs

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

**Review by whyrusleeping — APPROVED** [review]

File: cmd/lotus-shed/cid.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+var cidIdCmd = &cli.Command{
+	Name:  "id",
+	Usage: "create identitg CID from hex or base64 data",
+	Action: func(cctx *cli.Context) error {
+		if !cctx.Args().Present() {
+			return fmt.Errorf("must specify data")
+		}
+
+		dec, err := hex.DecodeString(cctx.Args().First())
```
Comment (whyrusleeping) [inline cmd/lotus-shed/cid.go:29]: technically you can have a string be valid in both hex and base64, can't you?

Comment (jimpick) [reply cmd/lotus-shed/cid.go:29]: s/identitg/identity/

File: cmd/lotus-shed/cid.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+var cidCmd = &cli.Command{
+	Name: "cid",
+	Subcommands: cli.Commands{
+		cidIdCmd,
+	},
+}
+
+var cidIdCmd = &cli.Command{
+	Name:  "id",
+	Usage: "create identitg CID from hex or base64 data",
```
Comment (jimpick) [inline cmd/lotus-shed/cid.go:23]: s/identitg/identity/

File: cmd/lotus-shed/cid.go (hunk 1 above)
Comment (magik6k) [reply cmd/lotus-shed/cid.go:29]: Highly unlikely unless you specifically craft one

**Pull request merged**